pkg/cmd/admin/router/router.go

```diff
@@ -293,6 +299,15 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 					{Type: dapi.DeploymentTriggerOnConfigChange},
 				},
 				Template: dapi.DeploymentTemplate{
+					Strategy: dapi.DeploymentStrategy{
```

Comment:
Rolling is the default. 

 @ramr it may be cleaner to make the defaulter to check for the fields individually.  Right now it only defaults if the entire params object is not set which means if you want to set any parameter you won't get the defaults.  If we check individually you'd only have to specify your custom update %.  https://github.com/pweil-/origin/blob/791188aae822c428a7e77ea8d89cfee26218d21c/pkg/deploy/api/v1/defaults.go#L23-23

/cc @ironcladlou